readme.md

```diff
@@ -16,21 +16,29 @@ Enable Accelerated Mobile Pages (AMP) on your WordPress site.
+Some of the most salient features and capabilities provided bythe plugin are:
 
-Also, your pages and custom post types can have AMP versions. Simply check their boxes on the 'AMP Settings' page in `/wp-admin`.
+- **Compatibility tool**: to assist the development of AMP experiences by enabling AMP debugging based on exposing extensive and detailed information about validation errors that may exist, the markup/scripts causing them, and the specific components on site (e.g theme, plugin, core) bearing the responsibility of that page content.
+- **CSS Tree Shaking**: to assist in the process of putting the CSS-house in order in cases where  existing CSS rule exceed the maximum limited permitted on single AMP pages.
+- **Core theme’s support**: enabling full AMP validity for four default themes (i.e.  2015, 2016, 2017, 2019).
+- **Gutenberg support**: enabling AMP content creation fully integrated with Gutenberg, the new and powerful editing experience in WordPress.
+- **AMP-first experiences support**: enabling full-site AMP experiences without sacrificing by one-bit the flexibility of the platform, or the fidelity of content.
+- **Many Optimizations**: A myriad of code, performance, and developer experience improvements: from customization flexibility, to better UI flows, internationalization, accessibility, etc. Check the full list in the release post.
 
-Your entire site can render as "Native AMP" if your theme calls `add_theme_support( 'amp' )`. There will only be one version of each URL: the AMP version. There won't be separate URLs with `/amp` or `?amp` appended. See this [wiki page](https://github.com/Automattic/amp-wp/wiki/Adding-Theme-Support#native-amp) for details and restrictions.
+With the official AMP plugin for WordPress, the WordPress ecosystem is provided with the capabilities and tools it needs to build world-class AMP experiences without deviating from its standard, flexible,and well-known content creation workflow.
 
-Your theme can also use [Paired Mode](https://github.com/Automattic/amp-wp/wiki/Adding-Theme-Support#paired-mode), with your own custom templates for the AMP URLs.
+## Getting Started ##
 
-"Native AMP" and "Paired Mode" add full support for commenting and widgets.
+To learn more about the plugin, how to install it, and start leveraging its capabilities to power your AMP content creation workflow check [the official AMP pluging product site](htts://amp-wp.org).
 
-If your theme doesn't support `'amp'`, this will use basic legacy post templates for AMP consumers like Google Search and Twitter. And when visiting the site, the AMP content won't normally appear without appending strings to the URL like `/amp` or `?amp`.
+In a nutshell, follow these steps to get started quickly:
+1. Upload the folder to the `/wp-content/plugins/` directory
```

Comment:
Do we want to revisit these instructions? They seem very developer specific and not user focused.. If they are a user, most will install directly from the admin. I'm guessing we'd want to optimize for the _average_ user's installation process?